jrt/src/com/yahoo/jrt/Supervisor.java

```diff
@@ -33,6 +35,16 @@ public Supervisor(Transport transport) {
         new MandatoryMethods(this);
     }
 
+    /**
+     * Will optimize buffers size for small memory footprint
+     * Use this when you have many connections with very little traffic.
+     **/
+    public Supervisor enableSmallBuffers() {
```

Comment:
I think 'useSmallBuffers' would be a better name, since it does not enable the use of small buffers as much as it disables the use of large ones.

Reply:
Agree, done